app.js

```diff
@@ -57,7 +57,7 @@ const opts = nopt({
     port: [Number],
     propDebug: [Boolean],
     debug: [Boolean],
-    static: [String],
+    dist: [Boolean],
```

Comment:
See note in the compiler-explorer-image PR; that still passes `--static`

Reply:
addressed in other PR comments

Reply:
thanks!